### PR TITLE
Add source of wealth documentation

### PR DESCRIPTION
**Pull request by hugopeixoto**

### CODE REVIEW

File: kyc-levels.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Fractal ID supports different levels of verification. Additionally, each level m
 | `video` | Identity check using real time video call \(Video ID\). |
 | `accreditation` | Accreditated investor check; applicable to US and Canadian residents only. |
 | `wallet` | Crypto currency wallet check; uses [Coinfirm](https://www.coinfirm.com/). |
+| `sow` | Source of wealth check. |
```
Comment (ZilvinasKucinskas) [inline kyc-levels.md]: We should probably add `sow` to add-on matrix too, right?

Comment (hugopeixoto) [reply kyc-levels.md]: Yes, thanks for noticing. I grepped `ssn` but somehow missed that one

Comment (ZilvinasKucinskas) [reply kyc-levels.md]: Should we mention source of funds here too. Sales team is selling this as `source of funds` 🤔

File: user-integration/user-information-retrieval.md [hunk 2]
```diff
@@ -153,6 +153,45 @@ This endpoint returns JSON. The scopes associated with the access token will def
...
+| `sow_sale_of_property_bank_statement_file` | URL | `sow` | People only |
+| `sow_sale_of_property_contract_file` | URL | `sow` | People only |
+| `sow_sale_of_shares_bank_statement_file` | URL | `sow` | People only |
+| `sow_sale_of_shares_proof_file` | URL | `sow` | People only |
+| `sow_type` | `crypto`, `dividends`, `divorce_settlement`, `freelance`, `gifts`, `inheritance`, `lease_revenue`, `life_policy`, `pension`, `salary`, `sale_of_company`, `sale_of_property`, or `sale_of_shares` | `sow` | People only |
+| `sow_balance_sheet_file` | URL | `sow` | Institutions only |
+| `sow_cash_flow_statement_file` | URL | `sow` | Institutions only |
+| `sow_income_statement_file` | URL | `sow` | Institutions only |
+| `sow_unaudited_accountant_name` | URL | `sow` | Institutions only |
+| `sow_unaudited_accountant_professional_body` | URL | `sow` | Institutions only |
```
Comment (ZilvinasKucinskas) [inline user-integration/user-information-retrieval.md]: `string`?

File: user-integration/user-information-retrieval.md [hunk 3]
```diff
@@ -153,6 +153,45 @@ This endpoint returns JSON. The scopes associated with the access token will def
...
+| `sow_sale_of_company_contract_file` | URL | `sow` | People only |
+| `sow_sale_of_property_bank_statement_file` | URL | `sow` | People only |
+| `sow_sale_of_property_contract_file` | URL | `sow` | People only |
+| `sow_sale_of_shares_bank_statement_file` | URL | `sow` | People only |
+| `sow_sale_of_shares_proof_file` | URL | `sow` | People only |
+| `sow_type` | `crypto`, `dividends`, `divorce_settlement`, `freelance`, `gifts`, `inheritance`, `lease_revenue`, `life_policy`, `pension`, `salary`, `sale_of_company`, `sale_of_property`, or `sale_of_shares` | `sow` | People only |
+| `sow_balance_sheet_file` | URL | `sow` | Institutions only |
+| `sow_cash_flow_statement_file` | URL | `sow` | Institutions only |
+| `sow_income_statement_file` | URL | `sow` | Institutions only |
+| `sow_unaudited_accountant_name` | URL | `sow` | Institutions only |
```
Comment (ZilvinasKucinskas) [inline user-integration/user-information-retrieval.md]: `string`?

File: user-integration/user-information-retrieval.md [hunk 4]
```diff
@@ -153,6 +153,45 @@ This endpoint returns JSON. The scopes associated with the access token will def
...
+| `sow_dividends_bank_statement_file` | URL | `sow` | People only |
+| `sow_dividends_entitlement_file` | URL | `sow` | People only |
+| `sow_divorce_settlement_bank_statement_file` | URL | `sow` | People only |
+| `sow_divorce_settlement_proof_file` | URL | `sow` | People only |
+| `sow_freelance_bank_statement_file` | URL | `sow` | People only |
+| `sow_freelance_last_year_tax_returns_file` | URL | `sow` | People only |
+| `sow_freelance_service_contracts_file` | URL | `sow` | People only |
+| `sow_gifts_bank_statement_file` | URL | `sow` | People only |
+| `sow_gifts_name_of_donator` | URL | `sow` | People only |
+| `sow_gifts_reason_for_donation` | URL | `sow` | People only |
```
Comment (ZilvinasKucinskas) [inline user-integration/user-information-retrieval.md]: type == `string`?

File: user-integration/user-information-retrieval.md [hunk 5]
```diff
@@ -153,6 +153,45 @@ This endpoint returns JSON. The scopes associated with the access token will def
...
+| `sow_crypto_transactions_last_3_months_file` | URL | `sow` | People only |
+| `sow_dividends_bank_statement_file` | URL | `sow` | People only |
+| `sow_dividends_entitlement_file` | URL | `sow` | People only |
+| `sow_divorce_settlement_bank_statement_file` | URL | `sow` | People only |
+| `sow_divorce_settlement_proof_file` | URL | `sow` | People only |
+| `sow_freelance_bank_statement_file` | URL | `sow` | People only |
+| `sow_freelance_last_year_tax_returns_file` | URL | `sow` | People only |
+| `sow_freelance_service_contracts_file` | URL | `sow` | People only |
+| `sow_gifts_bank_statement_file` | URL | `sow` | People only |
+| `sow_gifts_name_of_donator` | URL | `sow` | People only |
```
Comment (ZilvinasKucinskas) [inline user-integration/user-information-retrieval.md]: type == string?

File: kyc-levels.md (hunk 1 above)
Comment (juliosantos) [reply kyc-levels.md]: https://trustfractal.slack.com/archives/CEMTWQ411/p1561641856026800

**Pull request merged**